backend/endpoints/rooms.py

```diff
@@ -76,6 +77,88 @@ async def create_room(
     )
 
 
+@router.patch(
+    "/org/{org_id}/rooms/{room_id}/members/{member_id}",
+    response_model=ResponseModel,
+    status_code=status.HTTP_200_OK,
+    responses={
+        404: {"detail": "room or member not found"},
+        424: {"detail": "member removal unsuccessful"},
+    },
+)
+async def remove_member(
+    org_id: str, room_id: str, member_id: str, admin_id: Optional[str] = None
+):
+    """Removes a member from a room either when removed by an admin or member leaves the room.
+
+    Fetches the room which the member is removed from from the database collection
+    Pops the member being removed from the room's members dict
+    Updates the database collection with the new room
+    Returns the room dict if member was removed successfully
+
+    Args:
+        org_id (str): A unique identifier of an organisation
+        member_id (str): A unique identifier of the member being removed from the room
+        room_id (str): A unique identifier of the room a member is being removed from
+        admin_id (str): A unique identifier of the member removing another member
+
+    Returns:
+        HTTP_200_OK (member removed from room): {room}
+    Raises
+        HTTP_404_NOT_FOUND: room or member not found
+        HTTP_403_FORBIDDEN: not authorized to remove room  member
+        HTTP_424_FAILED_DEPENDENCY: member removal unsuccessful
+    """
+    room_obj = await get_room(org_id, room_id)
```

Comment:
Let's make it more explicit: 
```suggestion
    room_data = await get_room(org_id, room_id)
```

Reply:
done thanks